src/components/home/card.js

```diff
@@ -1,50 +1,50 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import PropTypes from 'prop-types';
-import { removeBook } from '../../redux/books/booksSlice';
+import { v4 as uuidv4 } from 'uuid';
+import { getBooks, removeBook } from '../../redux/books/booksSlice';
 
 const CardContainer = () => {
-  const books = useSelector((state) => state.books.books);
-  return (
-    <div className="card-container">
-      {books.map((book) => (<Card book={book} key={book.title} />))}
-    </div>
-  );
-};
-
-const Card = (props) => {
-  const { book } = props;
+  const { books, isLoading, error } = useSelector((state) => state.books);
   const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  if (error) {
+    return <p>Error occurred</p>;
+  }
   return (
-    <div className="card">
-      <ul>
-        <li>
-          Book:
-          {book.title}
-        </li>
-        <li>
-          Author:
-          {book.author}
-        </li>
-      </ul>
-      <button
-        type="button"
-        onClick={() => {
-          dispatch(removeBook(book.item_id));
-        }}
-      >
-        Remove book
-      </button>
+    <div className="card-container">
+      {books.map((book) => (
+        <div className="card" key={uuidv4()}>
+          <ul>
+            <li>
+              num item:
```

Comment:
[OPTIONAL] This text could be `Name` or `Title` to avoid any confusion on what should be here 😉